### PR TITLE
Svg support

### PR DESCRIPTION
**Pull request by ethanclevenger91**
This update adds a filter to WordPress to allow SVG uploads, a dropdown under "Current Menu Settings" to swap between raster images and SVG images, and the vanilla URL to the selected var for the preview pane since SVGs don't have "sizes".

Did not update readme or screenshots

### CODE REVIEW

**Comment by kucrut** [comment]
@ethanclevenger91 Could you please refresh this PR? I'll gladly include this feature in the next version. Also, please don't commit changes to the minified CSS/JS. We'll take care of that when releasing a new version.

If you don't think you'll have the time, I'll adapt your changes in a new PR. Just let me know :)

**Comment by ethanclevenger91** [comment]
Holy Travis Batman. Is there a better way of running these tests locally before pushing changes? Ready for merge I do believe.

File: includes/type-image.php [hunk 1]
```diff
@@ -255,16 +255,21 @@ protected function add_icon( $title, $values ) {
 		if ( ! empty( $i_style ) ) {
 			$i_attrs['style'] = $i_style;
 		}
-
-		$title = sprintf(
-			'%s%s%s',
-			'before' === $values['position'] ? '' : $title,
-			wp_get_attachment_image(
+		$mime_type = get_post_mime_type( $icon->ID );
+		if ( 'image/svg' == $mime_type || 'image/svg+xml' == $mime_type ) {
```
Comment (kucrut) [inline includes/type-image.php]: @ethanclevenger91 Please use strict comparison `===`.

**Comment by kucrut** [comment]
This is looking good. I'll test this out tomorrow.

One more cleanup: Please revert the changes in `languages/menu-icons.pot` and `*.min.css` (you can checkout those files from the `develop` branch).

To run the tests locally, you need to have PHPCS, JSCS & JSHint setup and use the [pre-commit hook](https://github.com/xwp/wp-dev-lib/#pre-commit-hook).

**Comment by kucrut** [comment]
I've tested this branch, and it seems there are some functionalities missing (preview, size settings, etc). I'm closing this PR and created [a new PR](https://github.com/kucrut/wp-menu-icons/pull/50) for the SVG support with your changes as a base. Please feel free to issue PR(s) to the `svg-support` branch if you'd like to contribute.

Thanks!

**Pull request closed without merging**